calculator/urls.py

```diff
@@ -3,6 +3,11 @@
 
 
 urlpatterns = [
+    url(
+        r'^v1/clients/$',
```

Comment:
The regex pattern, used in url() function for url pattern is outdated in django. We could use path() instead, like this:
path('client/', views.post_clients, name='post_clients')